packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -119,30 +146,50 @@ const hasValidParticipant = (chatMessage: ChatMessageWithStatus): boolean =>
  * @public
  */
 export const messageThreadSelector: MessageThreadSelector = createSelector(
-  [getUserId, getChatMessages, getLatestReadTime, getIsLargeGroup],
-  (userId, chatMessages, latestReadTime, isLargeGroup) => {
+  [getUserId, getChatMessages, getLatestReadTime, getIsLargeGroup, getReadInformation, getParticipants],
+  (userId, chatMessages, latestReadTime, isLargeGroup, readInformation, participants) => {
```

Comment:
I would still name this as readReceipt cause it would be good to reflect the type using name(unless it is not possible) - I see why you give it a new name because you have another variable named the same - 
I would say 2 ways of naming - 
1. name parameter this as allReadReceipts, then keep the second one
2. name this parameter as readReceipts then rename the variable as readReceiptForMessage

Personally I prefer second one cause your basic selector getReadInformation(which might be renamed as well) would be reused in the future - so name it as getReadReceipts sounds more readable